doc-content/enterprise-only/optimizer/ex-cloudBalancing-ref.adoc

```diff
@@ -1,4 +1,4 @@
 [id='ex-cloudBalancing-ref']
 = Cloud balancing
 
-For information about this example, see <<cloudbal-tutorial-con>>.
\ No newline at end of file
+For information about this example, see {URL_GETTING_STARTED}#cloudbal-tutorial-con[_Getting started with Java solvers: A cloud balancing example_].
```

Comment:
Broken link but I assume that is because the 7.11 version of the tutorial doc is not live yet.